website/src/markdownPlugins/publishedProperties.ts

```diff
@@ -34,22 +34,29 @@ export class PublishedProperties extends HTMLWidget {
             const w = new mod[widget]();
             const derivedFrom = this.extends(w);
             md.push(`Derived from:  ${derivedFrom[1]} (${derivedFrom[0]})\n`);
+            md.push("Published properties act as both getter and setter methods. If a value is passed as an argument the property will be set and the widgets' context will be returned allowing these properties to be chained as seen in the above code.\n");
             const pp: Meta[] = w.publishedProperties(false, true);
+            md.push("|Property|Type|Optional?|Default|Description|");
+            md.push("|--------|----|---------|-------|-----------|");
             pp.forEach(meta => {
-                md.push(`### ${meta.id}\n`);
-                md.push(`_${meta.description}_\n`);
-                md.push(`* **type**: ${meta.type}`);
-                md.push(`* **optional**: ${!!meta.ext.optional}`);
-                md.push(`* **default**: ${JSON.stringify(meta.defaultValue)} `);
+                let setStr = "";
                 if (meta.type === "set") {
                     try {
                         const set = typeof meta.set === "function" ? meta.set() : meta.set;
-                        md.push(`* **options**: "${set.join('" | "')}" `);
+                        setStr = `<br/> <i style="font-weight:bold;">Potential values include:</i> "${set.join('", "')}"`;
```

Comment:
Do you need html formatting here?

Reply:
Yes, just to force a line break and style the font a bit.

Reply:
Can't you use "_**" to do the italic and bold?  (and \n for new line?)

Reply:
The `\n` causes it to be treated as if it were a new table row.

Reply:
I've changed the bold and italic segments to markdown, but the `<br>` is still needed for the line break.